ajax/libs/jsrsasign/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jsrsasign",
-  "version": "7.1.5",
+  "version": "7.1.3",
```

Comment:
Yes, because we remove `7.1.5` files in this commit, and it cause the package.json modified.
And related changes should in the same commit.